torch/ao/quantization/_dbr/utils.py

```diff
@@ -236,6 +236,26 @@ def pack_weights_for_functionals(
     for _, child in module.named_children():
         pack_weights_for_functionals(child)
 
+def attach_scale_zp_values_to_model(
+    module: torch.nn.Module,
+) -> None:
+    """
+    Calculates the scale and zero_point from each observer and attaches
+    these values to the parent module. This is done to avoid recalculating
+    these values at inference.
```

Comment:
Could you clarify what you mean by recalculating the values at inference? in the previous flow were we calling calculate_qparams each time we do inference?
IIUC, it looks like this function gets called during convert

Reply:
> in the previous flow were we calling calculate_qparams each time we do inference?

yes, in the original prototype the scale and zero point were calculated from observers during inference, this PR makes it work the same way our other flows work